src/scenes/home/thanks/thanks.js

```diff
@@ -1,12 +1,32 @@
 import React, { Component } from 'react';
```

Comment:
this was for id.me integration.  We don't want this here, it will conflict with the branch for id.me integration.